lms/djangoapps/courseware/views/views.py

```diff
@@ -283,8 +283,9 @@ def course_info(request, course_id):
             # redirect to the dashboard page.
             if isinstance(access_response, StartDateError):
                 start_date = strftime_localized(course.start, 'SHORT_DATE')
```

Comment:
Yeah that's only in case when we need to encode a string for url.
Thanks for explanation.